biolink-model.yaml

```diff
@@ -9702,6 +9734,59 @@ classes:
+        range: DirectionQualifierEnum
+      object direction qualifier:
+        description: Either increased or decreased.
+        range: DirectionQualifierEnum 
+      predicate:
+        description: The only acceptable value is "associated_with" currently.
+        ifabsent: associated with
+      object: 
+        description: A state variable that is affected by either another state variable or process.
+      subject derivative qualifier: 
+        description: A property (p) which composes with the subject such that it becomes "p of subject." For instance, "abundance of phragmites australis" would have the subject "phragmites australis" and subject derivative qualifier "abundance"
+      object derivative qualifier: 
+        description: A property (p) which composes with the object such that it becomes "p of object." For instance, "abundance of phragmites australis" would have the subject "phragmites australis" and object derivative qualifier "abundance"
+      subject part qualifier:
+        description: >-
+          This property is primarily used to provide ingredients for a relational state variable (i.e. a variable whose definition depends on the other elements to which it refers). For instance, "phylogenetic distance" can be said to be a state variable of interest, but it only has meaning when applied to two or more taxa. In that case, one could supply two values as part qualifiers: "phragmites australis" and "typha latifolia". This would result in the subject being interpreted as "phylogenetic distance between phragmites australis and typha latifolia." Note that there are no directional constraints on this - it could also be read as "phylogenetic distance between typha latifolia and phragmites australis." This must have 0 or at least 2 entries.
+        multivalued: true
+        minimum_cardinality: 2
+      object part qualifier:
+        description: >-
+          This property is primarily used to provide ingredients for a relational state variable (i.e. a variable whose definition depends on the other elements to which it refers). For instance, "phylogenetic distance" can be said to be a state variable of interest, but it only has meaning when applied to two or more taxa. In that case, one could supply two values as part qualifiers: "phragmites australis" and "typha latifolia". This would result in the subject being interpreted as "phylogenetic distance between phragmites australis and typha latifolia." Note that there are no directional constraints on this - it could also be read as "phylogenetic distance between typha latifolia and phragmites australis." This must have 0 or at least 2 entries.
+        multivalued: true
+        minimum_cardinality: 2
+      conditioned by qualifier:
+        description: >-
+          This property applies to the qualified association and either upregulates or downregulates it. It can accept any combination of 0 to 2 state variables and processes.
```

Comment:
I'm curious how this relates to the `regulates` predicate: https://github.com/biolink/biolink-model/blob/ac34f0b1f0c9a86ecebc605eb738a2d632cb0fd3/biolink-model.yaml#L2820 
and the `upregulated` direction qualifier?  Would you happen to have an example `eenvironmental process association` with all fields filled out that I can look at? (or in paragraph form/plain text)